source/brailleTables.py

```diff
@@ -560,7 +560,10 @@ def listTables():
 # Translators: The name of a braille table displayed in the
 # braille settings dialog.
 addTable("zhcn-cbs.ctb", _("Chinese common braille (simplified Chinese characters)"), input=False)
-
+# Translators: The name of a braille table displayed in the
+# braille settings dialog.
+# This should be translated to '中文中国汉语现行盲文（无声调）' in Mandarin.
+addTable("zh-chn.ctb", _("Chinese (China, Mandarin) Current Braille System (no tones)"))

```

Comment:
could this be moved to be adjacent to the other "Chinese (China, Mandarin)" tables?